content/docs/06.enterprise/03.auth-rbac-user-management/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Authentication, RBAC, & User Management
```

Comment:
I wonder if this can be simplified to user management or multi-user.

saying that, some parts of multi-user are across multiple sections (like audit logs) so could be confusing but makes a better URL path